guide/Guide/7.array.md

```diff
@@ -79,12 +79,12 @@ Let's execute this code and see what it returns.
 As you can array push has correctly registered separate animals.
 
 ## 3. Searching for values
-As you may know, array functions mostly work based on element indexes. 
-Generally, this kind of behavior is convenient, but we also need a way to get those indexes from existing arrays.
+As you may know, array functions mostly work based on element indices. 
```

Comment:
in this context, use indexes. to be related to the name index in function description
(they both correct)